src/test/java/TddWorldTest.java

```diff
@@ -127,17 +121,33 @@ public void testIntPower() {
 		assertEquals(8, tdd.calculatePower(2, 3));
 	}
 	
+	@Test
+	public void testPowerNull() {
+		TddWorld tdd = new TddWorld();
+		Integer base=null;
+		Integer exponent=null;
+		assertThrows(NullPointerException.class, () -> tdd.calculatePower(base,exponent));
+		assertThrows(NullPointerException.class, ()-> tdd.calculatePower(base,exponent));
+		}
+	
 	
 	@Test
 	@DisplayName("Test Reverse String.. ")
-	public void testReverse() {
+	public void testReverse1() {
 		TddWorld tdd = new TddWorld();
 		assertEquals("olleh", tdd.reverse("hello"));
+	}
+	@Test
```

Comment:
We can try to be consistent and use the same description approach as used at line 135

Reply:
Alright!